### PR TITLE
Update tooling pkgs and lint project

### PR DESCRIPTION
**Pull request by kaisermann**
#### What problem is this solving?

Update front-end tooling packages and lint/format the project.

#### How to test it?

n/a

#### Screenshots or example usage:

n/a

### CODE REVIEW

File: react/components/TranslatedTitle.tsx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { FormattedMessage } from 'react-intl'
 
 // The b-s token makes the `b` class be applied only on mobile devices.
 const TranslatedTile: FunctionComponent<TranslatedTileProps> = ({ title }) => {
-  const titleText = title || <FormattedMessage id="base.title" />
+  const titleText = title ?? <FormattedMessage id="base.title" />
```
Comment (kaisermann) [inline react/components/TranslatedTitle.tsx:6]: If `titleText` can be empty, we should keep the `||` @lucasecdb

Comment (lucasecdb) [reply react/components/TranslatedTitle.tsx:6]: I think we can keep it with the nullish coalescing

File: react/package.json [hunk 2]
```diff
@@ -24,15 +25,12 @@
...
     "apollo-client": "^2.6.3",
-    "eslint": "^5.15.1",
-    "eslint-config-vtex": "^10.1.0",
-    "eslint-config-vtex-react": "^4.1.0",
-    "prettier": "^1.18.2",
     "react-dom": "^16.8.6",
-    "tslint-eslint-rules": "^5.4.0",
-    "typescript": "3.7.3"
+    "typescript": "3.7.3",
+    "vtex.render-runtime": "https://browserslist--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.render-runtime@8.111.0+build1594327796/public/@types/vtex.render-runtime",
```
Comment (lucasecdb) [inline react/package.json:32]: can you update this? the build is failing because of it

Comment (lucasecdb) [reply react/package.json]: done in 9c1c20f

**Pull request merged**